index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Mark Portfolio</title>
+</head>
+<body>
+    <header class="header-logo">
+        <h2 class="LOGO">Mark</h2>
```

Comment:
Kindly wrap your logo in an `<a>` tag🙏. Typically, the logo links to the homepage of a website. This action helps improve user experience

Reply:
Resolved it